projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.ts

```diff
@@ -99,13 +100,21 @@ export class QuotesComponent implements OnInit, OnDestroy {
   ) {
     this.selectedQuote = selectedQuote;
     const index = this.quoteCodesForCompare.indexOf(selectedQuote.cartCode);
-    if (checked && index === -1 && this.quoteCodesForCompare.length < 2) {
+    if (
+      checked &&
+      index === -1 &&
+      this.quoteCodesForCompare.length < QUOTE_COMPARISON_NUMBER
+    ) {
       this.quoteCodesForCompare.push(selectedQuote.cartCode);
     } else {
       this.quoteCodesForCompare.splice(index, 1);
     }
     this.disableCheckboxes(quotes);
-    if (this.quoteCodesForCompare?.length === 2) {
+    if (
+      checked &&
+      (this.quoteCodesForCompare?.length > 1 ||
+        this.quoteCodesForCompare?.length === QUOTE_COMPARISON_NUMBER)
```

Comment:
Message for successfully selected quotes will be displayed when there is a minimum 2 or equal to `QUOTE_COMPARISON_NUMBER` number of quotes. Also only when the checkbox is in the state `checked`, when it is unselected, the message won't be displayed.

Reply:
Can this be somehow united with the `if` above, since it's checking for the same conditions?

Reply:
No, we can't. The only same condition is `checked`, nothing else.